tasks/check_primary_cert.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+declare PT__installdir
+# shellcheck disable=SC1090
```

Comment:
SC1090 is disabled in most files. Let's disable it globally in CI. `SHELLCHECK_OPTS="--exclude SC1090"` to reduce the redundant disabling of it. This is probably a separate PR, but shellcheck can be added to the travis-ci.yml like the following. 

https://github.com/puppetlabs/support-tasks/blob/master/.sync.yml#L25-L34
https://github.com/puppetlabs/support-tasks/blob/master/.sync.yml#L48-L49
https://github.com/puppetlabs/support-tasks/blob/master/.sync.yml#L59-L62

Then run `pdk update` to sync the changes to the travis file.

Reply:
Are you ok with opening a new issue for setting up and configuring `shellcheck`?

Reply:
Yup